core/src/main/java/bisq/core/presentation/SupportTicketsPresentation.java

```diff
@@ -42,6 +43,11 @@
     @Getter
     private final BooleanProperty showOpenMediationTicketsNotification = new SimpleBooleanProperty();
 
+    @Getter
+    private final StringProperty numOpenRefundTickets = new SimpleStringProperty();
+    @Getter
+    private final BooleanProperty showOpenRefundTicketsNotification = new SimpleBooleanProperty();
```

Comment:
Are `numOpenRefundTickets` and `showOpenRefundTicketsNotification` variables used somewhere outside of `SupportTicketsPresentation` ?

Reply:
Not that I can see, just like the other properties.

Reply:
> Are `numOpenRefundTickets` and `showOpenRefundTicketsNotification` variables used somewhere outside of `SupportTicketsPresentation` ?

But yes, the getters for all properties are not necessary anymore (there was a use case in the past). I'll change it in one of my next PRs

Reply:
So why we need these properties ? Seems like we only set values, but never use them. Or I miss something ?

Reply:
> So why we need these properties ? Seems like we only set values, but never use them. Or I miss something ?

I changed it already locally where all except of on property are removed